ui/src/Elements/Message/README.md

```diff
@@ -0,0 +1,59 @@
+# Message
```

Comment:
This is on me as I forgot to add it as a property (appologies), but can we also have a 'isSelected' property as well? The purpose of this is to apply a highlight to the message on the interaction function being called. The use case is that when someone hovers (say) over offset 4 partion 1 being produced, the equivelent consumed message (if rendered) is highlighted also.

What I propose to implement this is the function to say I am selected/hovered is provided via onInteraction, and then the producer/consumer components set the isSelected property if it is rendering that message.

Thoughts?

Reply:
Nice idea - have added the `isSelected` property which adds a class when set.